index.css

```diff
@@ -235,13 +246,19 @@ a {
 /* -------------------------------------------- */
 
 .inspiration-section {
-    padding: 131px 88px 0 88px;
+    margin: 131px 88px 144px 88px;
```

Comment:
131px?

Reply:
Ouais, c'est pas vraiment équivalent partout...quand j'avais demandé à Raff il m'avait de bien respecter la landing as is. C'est une des notes que j'ai de mon côté que j'allais te partager à notre meeting.